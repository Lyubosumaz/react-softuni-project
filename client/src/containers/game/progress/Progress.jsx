import React from 'react';

export default function Progress() {
    return (
        <div >
            <h1>Progress</h1>

            <p>In progress... :(</p>
        </div>
    );
};