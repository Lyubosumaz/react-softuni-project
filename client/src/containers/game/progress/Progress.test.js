import React from 'react';
import { shallow } from 'enzyme';
import Progress from './Progress';

it('should renter Progress Component', () => {
    const wrapper = shallow(<Progress />);
    expect(wrapper).toBeTruthy();
});

it('should checks for html changes', () => {
    const wrapper = shallow(<Progress />);
    expect(wrapper).toMatchSnapshot();
});
