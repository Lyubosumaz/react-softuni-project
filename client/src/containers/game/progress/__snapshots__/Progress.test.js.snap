// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should checks for html changes 1`] = `
<div>
  <h1>
    Progress
  </h1>
  <p>
    In progress... :(
  </p>
</div>
`;
